docs/guis/chroma/gui.md

```diff
@@ -0,0 +1,182 @@
+# The Non-Linear Chromaticity GUI
+
+The non-linear chromaticity GUI provides functionality to analyze [chromaticity scans](#performing-scans) in the LHC.
+The GUI itself has no functionality for making these measurements and only serves to extract and analyze data from `NXCALS`.
+
+!!! info "Kerberos"
+    Note that using the GUI requires a valid Kerberos token.
+
+The GUI is published with `acc-py`, and can be run with:
+
+```bash
+/acc/local/share/python/acc-py/apps/acc-py-cli/pro/bin/acc-py app run chroma-gui
+```
+
+## Performing Scans
+
+Chromaticity scans are performed by modulating the frequency of the RF cavities in the LHC.
+The tune will be impacted by the chromaticity of the machine, and online tune data is automatically recorded to `NXCALS`.
+
+!!! warning "Feedbacks OFF"
+    Make sure to turn off both the tune and orbit feedbacks before performing an RF scan.
+    See with the current EIC that no other operations are ongoing that could be affected.
+
+To perform a scan, trim the RF circuit in steps to reach the desired momentum deviation.
+A good rule of thumb, _for the LHC_, is that:
```

Comment:
I guess this could use an how-to on these trims as well. Maybe a screenshot from the LSA gui? (or where this is done?)

Reply:
(Happy with it being noted as an issue todo in the near future)